### DISCUSSION TITLE
Instructions on rollups for Task/Event/User objects

### DISCUSSION THREAD

**Discussion by baobao917** [jamessimone/apex-rollup#511] [Q&A] [answered]
Suggestions for the README/Wiki:

- Section to specifically highlight how to perform rollups on these objects.  I remember seeing mention of it the first time I read the documentation but had difficulty finding it when I needed to.  Also it appears to have parts of information dispersed throughout.
- Some extra detail on how to perform rollups on these objects.  It states you can't use CMDT but ... "you can use either the Invocable or static methods exposed on Apex Rollup from Apex to roll values up."  I'm not sure exactly what this means.  I tried to find reference to this in other sections but didn't find any additional detail.

To that end, could you explain how to perform rollups on these objects (specifically Task object for my use-case scenario)?

![image](https://github.com/jamessimone/apex-rollup/assets/104730171/eed5cfdb-c47e-4b9a-83cb-93fe0b6f029d)

**Comment by jamessimone** [comment]
@baobao917 I'll be updating that section of the README based on your feedback here with a (hopefully) clearer explanation, including YouTube links to the exact timestamps where I show off this functionality

**Reply by baobao917** [reply, marked as answer]
@jamessimone Thanks.  I think I found it buried in the Rollup Custom Metadata Field Breakdown section.  Still a bit hard to find when specifically looking for it, but I do see it now and will review the information.  I am pasting screenshot here for others to reference where it is located.

![image](https://github.com/jamessimone/apex-rollup/assets/104730171/2173227a-d796-4262-b406-1fafdb29f0b0)
![image](https://github.com/jamessimone/apex-rollup/assets/104730171/14749665-abaf-47f9-9d7b-bf8fa9743b58)

**Reply by jamessimone** [reply]
I'll further enhance the visibility for these specific rollup types in the next README version

**Reply by baobao917** [reply]
@jamessimone Just one suggestion is that on the link to the Base action YouTube, it might be helpful to have it go directly to the 17m:10s mark or reference that timestamp in the README. Right now it's going to an early part of the video which isn't related to this topic.

**Comment by jamessimone** [comment]
Interesting, I will update that. It should have been the exact time stamp from that section, sorry about that On Oct 3, 2023, at 11:25 PM, Bao D ***@***.***> wrote:﻿
@jamessimone Just one suggestion is that on the link to the Base action YouTube, it might be helpful to have it go directly to the 17m:10s mark or reference that timestamp in the README. Right now it's going to an early part of the video which isn't related to this topic.
